graphics/graphviz/Portfile

```diff
@@ -30,17 +22,23 @@ long_description                Graphviz is ${description}. Graph visualization
                                 simple text language, and produce output in a \
                                 variety of visual and text formats.
 
-checksums                       rmd160  8fc103fc519e07e9db522ec2f748838571c71acf \
-                                sha256  ca5218fade0204d59947126c38439f432853543b0818d9d728c589dfe7f3a421
+use_parallel_build              yes
+use_bzip2                       no
+use_xz                          yes
+
+master_sites                    ${gitlab.homepage}/-/package_files/${package_id}/download?dummy=
```

Comment:
There's no need to declare `master_sites` anymore, as the gitlab portgroup takes care of it.

Reply:
This downloads a different tarball than the standard gitlab release workflow. See https://trac.macports.org/ticket/62165#comment:12 .